tests/plugins/slow_start.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+"""This plugin is used to check that updated connection hints work properly.
+
+"""
+from pyln.client import Plugin
+
+import socket
+import time
+
+plugin = Plugin()
+
+
+@plugin.init()
+def init(configuration, options, plugin):
+    plugin.keep_alive = True
+
+
+@plugin.async_method('waitconn')
+def wait_connection(request, plugin):
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    addr = ('localhost', 6669)
```

Comment:
Use port number 0, which will assign an unused ephemeral port?  Then use getsockname and print out what port you got.

This avoids conflicts on port 6669.

Reply:
nice, updated.